components/fxa-client/ios/FxAClient/FirefoxAccount.swift

```diff
@@ -170,6 +170,22 @@ open class FirefoxAccount {
         })
     }
 
+    open func getManageAccountURL(entrypoint: String) throws -> URL {
+        return try queue.sync(execute: {
+            return URL(string: String(freeingFxaString: try FirefoxAccountError.unwrap({err in
+                fxa_get_manage_account_url(self.raw, entrypoint, err)
+            })))!
+        })
+    }
+
+    open func getManageDevicesURL(entrypoint: String) throws -> URL {
```

Comment:
Do we return `URL` for other things in FxA?

Reply:
Yes, for example `getTokenServerEndpointURL` returns `URL`.